scripts/build/configure/termux_step_configure.sh

```diff
@@ -2,10 +2,42 @@ termux_step_configure() {
 	[ "$TERMUX_PKG_METAPACKAGE" = "true" ] && return
 
 	if [ "$TERMUX_PKG_FORCE_CMAKE" = "false" ] && [ -f "$TERMUX_PKG_SRCDIR/configure" ]; then
+		if [ "$TERMUX_CONTINUE_BUILD" == "true" ]; then
+			return;
+		fi
 		termux_step_configure_autotools
-	elif [ -f "$TERMUX_PKG_SRCDIR/CMakeLists.txt" ]; then
-		termux_step_configure_cmake
+	elif [ "$TERMUX_PKG_FORCE_CMAKE" = "true" ] || [ -f "$TERMUX_PKG_SRCDIR/CMakeLists.txt" ]; then
+		if [ "$TERMUX_CONTINUE_BUILD" == "true" ]; then
+			termux_setup_cmake
+			if [ "$TERMUX_CMAKE_BUILD" = Ninja ]; then
+				termux_setup_ninja
+			fi
+			return;
+		fi
+		termux_setup_cmake
```

Comment:
Move this up top, you can coalesce this whole top part like so:
```
termux_setup_cmake
if [ "$TERMUX_CMAKE_BUILD" = Ninja ]; then
  termux_setup_ninja
fi
if [ "$TERMUX_CONTINUE_BUILD" == "true" ]; then
  return;
fi
```

Reply:
Thanks, much neater!